core/src/test/scala/org/genivi/sota/core/ImpactResourceSpec.scala

```diff
@@ -42,16 +38,17 @@ class ImpactResourceSpec
     }  yield (pkg, device)
 
     val (pkg, device) = f.futureValue
+    val installedPkgs = Seq(pkg.id)
 
-    val affected = Map(device.uuid -> Seq(pkg.id))
+    deviceRegistry.setInstalledPackages(device.uuid, Seq(pkg.id))
```

Comment:
Yes, I believe so. It was using a fake resolver before, but I decided it would be nicer to just use the implementation in the fake device registry.